architecture/additional_concepts/storage.adoc

```diff
@@ -133,6 +133,29 @@ the volume after it is released. A PV's reclaim policy can be either `Retain` or
 Dynamically provisioned volumes have a default `ReclaimPolicy` value of `Delete`. Manually provisioned volumes have a default `ReclaimPolicy` value of `Retain`.
 ====
 
+
+[[reclaimmanual]]
+=== Manually reclaim a PV
+
+When a PersistentVolumeClaim is deleted, the PersistentVolume still exists and is considered "released". But the PV is not yet available for another claim because the previous claimant's data remains on the volume.
+
+To manually reclaim the PV as a cluster administrator:
+
+. Delete the PV.
++
+[source,terminal]
+----
+$ oc delete <pv-name>
+----
++
+The associated storage asset in the external infrastructure, such as an AWS EBS, GCE PD, Azure Disk, or Cinder volume, still exists after the PV is deleted.
+
+. Manually clean up the data on the associated storage asset.
+
+. Manually delete the associated storage asset, or if you want to reuse the same storage asset, create a new PV with the storage asset definition.
```

Comment:
Suggest splitting this into two shorter sentences:

Manually delete the associated storage asset. To reuse the same storage asset, create a new PV with the storage asset definition.

Reply:
Used your suggestion but added "alternately"

> Delete the associated storage asset. Alternately, to reuse the same storage asset, create a new PV with the storage asset definition.